presidio-analyzer/presidio_analyzer/predefined_recognizers/ner/gliner_recognizer.py

```diff
@@ -35,6 +40,9 @@ def __init__(
         multi_label: bool = False,
         threshold: float = 0.30,
         map_location: str = "cpu",
+        chunk_size: int = 250,
+        chunk_overlap: int = 50,
+        text_chunker: Optional[BaseTextChunker] = None,
```

Comment:
Should the chunk_size and chunk_overlap be parameters for the BaseTextChunker? Otherwise the user might get confused - they would pass a text_chunker and a chunk_size, but the chunk_size would not be used (as the text_chunker would have a different chunk_size). To me it feels that passing a text chunker is sufficient, and if someone wants to change chunking parameters, they can configure their text chunker differently. WDYT?